src/components/collectibles/CollectiblesHeader.jsx

```diff
@@ -0,0 +1,78 @@
+import { h } from 'preact'
+import BN from 'bn.js'
+import cn from 'classnames'
+import AudiusLogoButton from '../button/AudiusLogoButton'
+import styles from './CollectiblesHeader.module.css'
+import IconArrowGrey from '../../assets/img/iconArrowGrey.svg'
+import IconVerified from '../../assets/img/iconVerified.svg'
+import IconBronzeBadge from '../../assets/img/iconBronzeBadge.svg'
+import IconGoldBadge from '../../assets/img/iconGoldBadge.svg'
+import IconPlatinumBadge from '../../assets/img/iconPlatinumBadge.svg'
+import IconSilverBadge from '../../assets/img/iconSilverBadge.svg'
+import { getAudiusURL } from '../../util/shareUtil'
+
+const badgeTiers = [
```

Comment:
this should also probably be in a util so we can pull it out and show balances across the other bedtime things in the future